internal/injector/aspect/advice/code/template.go

```diff
@@ -98,6 +116,15 @@ func (t *Template) CompileExpression(ctx context.Context, node *node.Chain) (dst
 // compile generates new source based on this Template and returns a cloned
 // version of minimally post-processed dst.Stmt nodes this produced.
 func (t *Template) compile(ctx context.Context, chain *node.Chain) ([]dst.Stmt, error) {
+	decls, err := t.compileTemplate(ctx, "_statements_", chain)
+	if err != nil {
+		return nil, err
+	}
+
+	return decls[0].(*dst.FuncDecl).Body.List, nil
```

Comment:
Is this safe ?

Reply:
We control the template, so we know exactly what it can produce at the top-level... So... yeah ^^